components/usage-api/usage/v1/billing.proto

```diff
@@ -23,3 +26,14 @@ message UpdateInvoicesRequest {
 
 message UpdateInvoicesResponse {
 }
+
+message GetLatestInvoiceRequest {
+  string team_id = 1;
+  string user_id = 2;
```

Comment:
Should this be a [oneof](https://developers.google.com/protocol-buffers/docs/proto3#oneof) field?

Reply:
updated to one of